src/pages/rooms/chats.tsx

```diff
@@ -27,30 +37,49 @@ export default function Chats({ roomId }: Props): ReactElement {
     return <LoadingSpinner />;
   }
 
-  const onSend = (e: React.FormEvent<HTMLFormElement>) => {
-    e.preventDefault();
-    socket.emit(Events.create, { msg: input, userId: user.id });
+  const onEdit = (chat: Chat, callback: Dispatch<SetStateAction<string>>,
+  ): void => {
+    socket.emit(Events.update, { id: chat.id, msg: chat.msg }, () =>
+      callback(''),
+    );
   };
 
+  const onCreate = (msg: string, callback: Dispatch<SetStateAction<string>>) => {
+    socket.emit(Events.create, { msg, userId: user.id }, () =>
+      callback(''));
+  };
+
+  const onCancel = () => {
+    return
+  }
+
+  const onReply = (chat: Chat) => {
+    setInput(chat.msg)
+  }
+
   return (
-    <div>
-      <div className="ui relaxed divided list">
+    <div className="ui container" style={{ maxHeight: '900px' }}>
+      <br></br>
```

Comment:
this element has no childs, can be closed immediately

Reply:
Done